### DISCUSSION TITLE
Why does ZFS not notice a missing spare?

### DISCUSSION THREAD

**Discussion by PhilipMay** [openzfs/zfs#14412] [Q&A] [answered]
Hi,
I am unsing OpenZFS on Linux (zfs-dkms 2.1.7).
I have a hot spare disk.
Now I pull out the hot spare disk.
I do a "scrub".

But ZFS continues to think the spare is still online...

Is there a way how ZFS notices / check if all is ok with the spares?

Many thanks...

**Comment by amotin** [comment, marked as answer]
May be this: https://github.com/openzfs/zfs/pull/14295
